shared/templates/shell_lineinfile/tests/quoted.fail.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+FILE_PATH={{{ PATH }}}
+PARAMETER={{{ PARAMETER }}}
+VALUE={{{ VALUE }}}
+NO_QUOTES={{{ NO_QUOTES }}}
+NO_QUOTES=$(echo "$NO_QUOTES" | tr '[:upper:]' '[:lower:]')
+
+if [ "$NO_QUOTES" == "true" ]; then
+    if [ -f "$FILE_PATH" ]; then
+        sed -i "/^\s*${PARAMETER}.*/Id" "$FILE_PATH"
+    else
+        mkdir -p $(dirname "$FILE_PATH")
+        touch "$FILE_PATH"
+    fi
+    echo "${PARAMETER}=\"${VALUE}\"" > "$FILE_PATH"
```

Comment:
Is the Bash variable expansion needed?
Could it you use the be Jinja variable directly?
```suggestion
    echo "{{{ PARAMETER }}}=\"{{{ VALUE }}}\"" > {{{ FILE_PATH }}}
```

Reply:
Not needed, I will update it.